Lib/Timer.cpp

```diff
@@ -33,7 +33,7 @@
 #include "Timer.hpp"
 
 #define DEBUG_TIMER_CHANGES 0
-#define MILLION 1000000
+#define MILLION (1 << 20)
```

Comment:
Huh? Maybe `INSTRUCTION_UNIT` or something?

Reply:
Ahah, OK - it's like megabytes or whatever. In which case either ignore me or something like `MEGA_INSTRUCTION`.

Reply:
I think it's just the MEGA from MEGA_INSTRUCTION; thanks for the suggestions!